keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -286,6 +293,10 @@ _This section must be completed when targeting beta graduation to a release._
   Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
   checking if there are objects with field X set) may be a last resort. Avoid
   logs or events for this purpose.
+  An operator can check if the `NodeOutOfServiceVolumeDetach` feature gate is
+  enabled and if there is an `out-of-service` taint on the shutdown node.
+  The usage of this feature requires the manual step of applying a taint
+  so the operator should be the one applying it.
```

Comment:
I don't think feature gate can be checked externally by any agent. But yes - an operator can indeed monitor for the taint on the node objects.